src/uct/ib/base/ib_md.c

```diff
@@ -1215,6 +1236,10 @@ ucs_status_t uct_ib_md_open_common(uct_ib_md_t *md,
 
     uct_ib_md_set_pci_bw(md, md_config);
 
+#if HAVE_IBV_DM
+    md->alloc_align = 2;
+#endif
```

Comment:
why needed?

Reply:
dm allocation length is being aligned, inspired by your PR:
![image](https://github.com/openucx/ucx/assets/19361875/0acb2ab5-a40a-4ad1-bd48-0da8b4b51052)